main/src/ca/uwaterloo/flix/language/phase/jvm/BackendType.scala

```diff
@@ -17,48 +17,42 @@
 package ca.uwaterloo.flix.language.phase.jvm
 
 import ca.uwaterloo.flix.language.ast.MonoType
+import org.objectweb.asm.Opcodes._
```

Comment:
In my opinion `Opcodes._` shouldn't be imported, just write `Opcodes.AASTORE` fx at each spot